server/src/views/frame.js

```diff
@@ -40,31 +84,24 @@ exports.Frame = React.createClass({
       <script src={ this.props.linkify("/js/parent-helper.js") } />
       { favicon }
       <div id="toolbar">
-        <div className="logged-out" id="login-widget">
-          <div id="login-dropdown" className="login-collapsed">
-            <div id="login-compact">
-              <img src="https://pageshotpages.appspot.com/static/default-profile.svg" id="login-drop" className="user-avatar" style={{ height: "20px", width: "20px" }} />
-            </div>
-            <div id="login-dropped">
-              <img src="https://pageshotpages.appspot.com/static/default-profile.svg" className="user-avatar" style={{ height: "50px", width: "50px" }} />
-              <div className="if-logged-in">
-                <span className="user-email">-</span>
-                <br />
-                <span id="logout">logout</span>
-              </div>
-              <div className="if-logged-out">
-                Log in:
-                <input id="log-in-email" placeholder="email" type="text" />
-              </div>
-            </div>
-          </div>
-        </div>
         <a className="main-link" href={ shot.url }>
+          { shot.docTitle }
```

Comment:
Just FYI, eventually we'll probably use shot.title which will pick from the best title (maybe shot.userTitle || shot.ogTitle || shot.docTitle)